src/server/schemas/cart.schema.ts

```diff
@@ -0,0 +1,17 @@
+import mongoose from 'mongoose';
+import { Cart } from '../../shared/models/cart.model.js'
+
+const { Schema, model } = mongoose
+
+const cartSchema = new mongoose.Schema<Cart>({
```

Comment:
Line 6: since Schema and model are declared in line 4 as mongoose, you can just type "Schema" instead of "mongoose.Schema" here.